README.md

```diff
@@ -2,47 +2,87 @@
+
 ```bash
-$ git clone https://github.com/iceberg-project/Seals.git
-$ module load cuda
-$ module load python3
-$ virtualenv iceberg_seals
-$ source iceberg_seals/bin/activate
-[iceberg_seals] $ export PYTHONPATH=<path>/iceberg_seals/lib/python3.5/site-packages
-[iceberg_seals] $ pip install . --upgrade
+$ pwd
+/home/username
+$ cd $SCRATCH                      # switch to your working space.
+$ mkdir Seals                      # create a directory to work in.
+$ cd Seals                         # move into your working directory.
+$ module load cuda                 # load parallel computing architecture.
+$ module load python3              # load correct python version.
+$ virtualenv seals_env             # create a virtual environment to isolate your work from the default system.
+$ source seals_env/bin/activate    # activate your environment. Notice the command line prompt changes to show your environment on the next line.
+[seals_env] $ pwd
+/pylon5/group/username/Seals
+[seals_env] $ export PYTHONPATH=<path>/seals_env/lib/python3.5/site-packages # set a system variable to point python to your specific code. (Replace <path> with the results of pwd command above.
+[seals_env] $ pip install iceberg_seals.search # pip is a python tool to extract the requested software (iceberg_seals.search in this case) from a repository. (this may take several minutes).
 ```
 
-From PyPi:
+Method #2 (Installing from source; recommended for developers only): 
+
 ```bash
+$ git clone https://github.com/iceberg-project/Seals.git
 $ module load cuda
 $ module load python3
-$ virtualenv iceberg_seals
-$ source iceberg_seals/bin/activate
-[iceberg_seals] $ export PYTHONPATH=<path>/iceberg_seals/lib/python3.5/site-packages
-[iceberg_seals] $ pip install iceberg_seals.search
+$ virtualenv seals_env
+$ source seals_env/bin/activate
+[seals_env] $ export PYTHONPATH=<path>/seals_env/lib/python3.5/site-packages
+[seals_env] $ pip install . --upgrade
 ```
 
 To test
 ```bash
+[iceberg_seals] $ deactivate    # exit your virtual environment.
+$ interact -p GPU-small            # request a compute node (this may take a minute or two or more).
+$ cd $SCRATCH/Seals             # make sure you are in the same directory where everything was set up before.
+$ module load cuda                 # load parallel computing architecture, as before.
+$ module load python3              # load correct python version, as before.
+$ source seals_env/bin/activate # activate your environment, no need to create a new environment because the Seals tools are installed and isolated here.
+[iceberg_seals] $ iceberg_seals.detect --help  # this will display a help screen of available usage and parameters.
+```
+## Prediction
+- Download a pre-trained model at: https://github.com/iceberg-project/Seals/tree/feature/README/models/Heatmap-Cnt/UnetCntWRN/UnetCntWRN_ts-vanilla.tar 
```

Comment:
We may have to change this path when it is merged.

Reply:
Changed to master since the model is already there.